### PR TITLE
MARP-1074 Several UI adaptations

### PR DESCRIPTION
**Pull request by tvtphuc-axonivy**

### CODE REVIEW

File: marketplace-ui/src/app/modules/product/product-detail/product-detail-information-tab/product-detail-information-tab.component.scss [hunk 1]
```diff
@@ -1,9 +1,11 @@
 .info-title {
-  padding-top: 2px;
+  padding-top: 10px;
```
Comment (vhhoang-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail-information-tab/product-detail-information-tab.component.scss:2]: Use  1 rem (also 10px) for this padding-top

**Comment by vhhoang-axonivy** [comment]
image
The styling of social media links in footer is not correct. Styling the parent <span> does not apply to the <i> tag. You must style the <i> with font-size: 1.8rem to match the design in figma.

**Comment by vhhoang-axonivy** [comment]
image
Each social media link has a gap of 24px. Right now each link has a padding of 0.5rem, which in total they have a gap of only 1rem = 10px.

**Comment by vhhoang-axonivy** [comment]
The gap between search bar and types filter should be 30px (3rem).

**Comment by vhhoang-axonivy** [comment]
image
The gap between the search bar and product list should be 72px (7.2rem).

**Comment by vhhoang-axonivy** [comment]
image
The gap between the title/intro container and the main section should be 72px (7.2rem).

**Comment by vhhoang-axonivy** [comment]
image
For screen with 767px width and below, make the page about section and contribute link section fall off each other (just like in mobile layout).

**Comment by vhhoang-axonivy** [comment]
image
The current button is too small compared to figma's, style it again so the two matche.

**Comment by vhhoang-axonivy** [comment]
image
image

The gap between the "< Back to overview" and the page header is too small. It should be the same just like in the home page, which is 82px (8.2rem).

**Comment by vhhoang-axonivy** [comment]
image
image
The gap between the content tab and rating section should be 96px (9.6rem). Right now they only have a gap of 3.2rem (32px).

**Comment by vhhoang-axonivy** [comment]
image
image

In mobile view, the dropdown tab content selection is too small. Make it the same as in figma.

**Comment by tvtphuc-axonivy** [comment]
> image
> 
> The gap between search bar and types filter should be 30px (3rem).

Done

**Comment by tvtphuc-axonivy** [comment]
> image The gap between the search bar and product list should be 72px (7.2rem).

Done

**Review by vhhoang-axonivy — APPROVED** [review]
LGTM

**Pull request merged**